contrib/google.golang.org/grpc.v12/grpc.go

```diff
@@ -41,7 +40,7 @@ func startSpanFromContext(ctx context.Context, method, service string) (ddtrace.
 		tracer.ServiceName(service),
 		tracer.ResourceName(method),
 		tracer.Tag("grpc.method", method),
-		tracer.SpanType("go"),
+		tracer.SpanType(ext.AppTypeRPC),
```

Comment:
is RPC the new name for "custom"?

Reply:
This is what we were using in most places and it was there. It still shows as custom in the UI. Maybe there was a plan for it to be added a separate section? No idea.

Reply:
there isn't

Reply:
Well, what today is a tab tomorrow could be a dropdown 😄 I think anything that doesn't fall into "web" or "db" will be marked as custom. It won't hurt being specific about it though in case we ever change our minds later.